models/modelsmodule.cpp

```diff
@@ -178,6 +182,10 @@ ModelsModule::init( SLIInterpreter* )
   kernel().model_manager.register_node_model< parrot_neuron >( "parrot_neuron" );
   kernel().model_manager.register_node_model< pp_psc_delta >( "pp_psc_delta" );
   kernel().model_manager.register_node_model< pp_pop_psc_delta >( "pp_pop_psc_delta" );
+  kernel().model_manager.register_node_model< gif_psc_exp >( "gif_psc_exp" );
+  kernel().model_manager.register_node_model< gif_psc_exp_multisynapse >( "gif_psc_exp_multisynapse" );
+  kernel().model_manager.register_node_model< gif_cond_exp >( "gif_cond_exp" );
+  kernel().model_manager.register_node_model< gif_cond_exp_multisynapse >( "gif_cond_exp_multisynapse" );
```

Comment:
The `cond` models, which use the `GSL` have to be encapsulated by

    #ifdef HAVE_GSL
    ...
    #endif
Otherwise the template argument is not valid, when you compile without `GSL`.